docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -164,3 +164,257 @@ The test results indicate that the minimum amount of memory for running {Product
 ** Installing the cluster monitoring tools requires an additional 5 GB of memory.
 ====
 
+== Red Hat Single Sign-On
+{ProductShortName} creates a basic
+https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6[Red
+Hat Single Sign-On] instance, using the Keycloak CR of Red Hat Single
+Sign-On Operator. Keycloak is a single sign-on solution for web apps and
+RESTful web services.
+
+The {ProductShortName} Keycloak realm is fully configured by the Hub.
```

Comment:
Refer to Keycloak only when dealing with the CR name. The rest should be replaced by Red Hat Single Sign-on